DirectEmployers/MyJobs
# Ensure that we don't show another partner's contacts when creating a partner [PD-2634]

## What's this PR do?
Stops a heroic filtering that ended up being silly
## Screenshots (if appropriate)

## Where should the reviewer start?
The three modified lines
## How should this be manually tested?
Go to NUO.
Begin creating a new partner through the NUO wizard
Ensure that existing contacts don't appear in the contact autocomplete
## Any background context you want to provide?
